systemvm/scripts/config_ssl.sh

```diff
@@ -123,6 +123,8 @@ customCACert=
 publicIp=
 hostName=
 keyStore=$(dirname $0)/certs/realhostip.keystore
+defaultJavaKeyStoreFile=/etc/ssl/certs/java/cacerts
+defaultJavaKeyStorePass=changeit
```

Comment:
Those are the defaults of the OS, so Debian in this case. The passwd is the default one from Java itself. I see no reason now why you should want that to be configurable. It is possible with a global settings, then passing that via cmd_line to the vm and read it here.